src/wp-includes/class-wp-theme-json.php

```diff
@@ -2736,6 +2765,20 @@ protected static function remove_insecure_settings( $input ) {
 				}
 			}
 		}
+
+		// Ensure indirect properties not included in any `PRESETS_METADATA` value are allowed.
+		foreach ( static::INDIRECT_PROPERTIES_METADATA as $property => $paths ) {
+			foreach ( $paths as $path ) {
+				$value = _wp_array_get( $input, $path, array() );
+				if (
+					null !== $value &&
+					! is_array( $value ) &&
```

Comment:
```suggestion
					is_string( $value ) &&
```
This check exists to ensure a string is passed to `WP_THeme_JSON::is_safe_css_declaration()`, which expects a string data type. Instead of checking for the absence of `null` or `array`, flip it around to check if it's a string. Why? To simplify the code and make it more clear as to what data type is expected.

Reply:
Resolved https://github.com/WordPress/wordpress-develop/pull/3742/commits/c05ca999b5875bb06eda819ce723d3adb1b42307